src/components/middle/immedia/Immedia.tsx

```diff
@@ -404,7 +404,8 @@ const Immedia: FC<OwnProps & StateProps> = ({ chatId, currentUser }) => {
               </video>
               <canvas
                 ref={canvasMeRef}
-                className="photo-canvas"
```

Comment:
If we remove the `photo-canvas` class, let's make sure we also remove it from the `scss` file as well.

Reply:
I did not delete it because it's used in the remaining canvas https://github.com/ekumenlabs/telegram-tt/blob/ac446a3894aa6c67b63ef3f0c2e86ebf1fab4b35/src/components/middle/immedia/Immedia.tsx#L419-L422